client/my-sites/plan-features/style.scss

```diff
@@ -1,3 +1,33 @@
+$header-banner-height: 20px;
```

Comment:
Good Point. Lets namespace the variable like `$plan-features-header-banner-height` or you can just use the bare value. Feel free to choose what you think works better.